src/SecretsService/Startup.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace ACMTTU.NoteSharing.SecretsService
+{
+    public class Startup

```

Comment:
I think we should be injecting the DB, HTTP Web Client, web client validation middleware and Neptune services at startup.

Reply:
Well... we have the chicken and the egg problem. Neither of our PRs are in so we I can't do that just yet

Reply:
But yes, when both our PR's are in, we should definitely inject the stuff we just built into all the services and applications

Reply:
Sry, getting ahead of myself. We can't the validation middleware, but we can the IHttpClientFactory and the database.